### PR TITLE
fix: show draft banner on all draft content (resolves #1299)

### LINKED ISSUE #764: Individual page creation: save and save next buttons placement
- [ ] Remove the set of Save, Save next, and Save previous from the top of the form. They make it very overwhelming for the magnifier users
- [ ] Place Save and Save next next to each other. Magnifier user missed the save next button

### LINKED ISSUE #920: Form fields: Structure of pages
For any pages that involve complex form fields with a step tracker (member page creation, project creation), make sure each page has the following structure:
- Title of the step is h2
- Underneath the title of the step is a 3px stroke divider
- at the bottom after the last form field, have a 3px stroke divider followed by the 2 CTA's (Save & save and next / save and previous)

### LINKED ISSUE #937: Individual page creation: "Save and next" styling
To minimize user movements, prioritize "Save and next" and use the primary button styling for it and make the "Save" secondary.

### LINKED ISSUE #1299: Previewing profile: draft mode notification doesn't appear in other tabs
**Prerequisites**

- [x] I've [searched](https://github.com/accessibility-exchange/platform/issues) for existing issues

**Describe the bug**
Responding to #1288 and #675 - the warning banner in preview mode is there, but if I click to another tab (ex. projects on FRO page), then back to the "about", the banner disappears.

**To Reproduce**
1. Make an FRO page, make sure account is verified
2. Fill in all required fields and preview the page
3. Click "projects" in the side bar

**Expected behavior**
The notification should be persistent as long as you are in previewing your page.

### PR DESCRIPTION
**Pull request by greatislander**
- [x] Resolves #1299

Bonus:

- [x] Resolves #764
- [x] Resolves #920 
- [x] Resolves #937

### Prerequisites

If this PR changes PHP code or dependencies:

- [x] I've run `composer format` and fixed any code formatting issues.
- [x] I've run `composer analyze` and addressed any static analysis issues.
- [x] I've run `php artisan test` and ensured that all tests pass.
- [x] I've run `composer localize` to update localization source files and committed any changes.

If this PR changes CSS or JavaScript code or dependencies:

- [x] I've run `npm run lint` and fixed any linting issues.
- [x] I've run `npm run build` and ensured that CSS and JavaScript assets can be compiled.

### CODE REVIEW

File: resources/lang/lsq.json [hunk 1]
```diff
@@ -93,7 +94,7 @@
     ":projects projects and :engagements engagements you are running": "Vous êtes en charge de :project projets et de :engagements consultations",
     ":role Invitation": "Invitation à devenir :role",
     ":service": ":service",
-    ":signLanguage (with :locale)": ":signLanguage (avec :locale)",
+    ":signLanguage (with :locale)": ":signLanguage (with :locale)",
```
Comment (jobara) [inline resources/lang/lsq.json]: This is unfortunate. I wonder if there's a bug in the copying. Would be a bit annoying if this needs to be manually corrected on every PR.

Comment (greatislander) [reply resources/lang/lsq.json]: I don't think it is a bug in the copying. I fixed it in fr.json and it copied correctly. I'm not sure what's up, though.

**Comment by jobara** [comment]
@greatislander it doesn't appear that the FRO's page has been updated to address #764 and although it's a single step, it might be more consistent to also add in the horizontal rules as mentioned in #920.

**Comment by greatislander** [comment]
@jobara done

**Review by jobara — APPROVED** [review]

**Pull request merged**